src/main/java/statisticker/Statistics.java

```diff
@@ -1,10 +1,32 @@
 package statisticker;
 
-import java.util.List;
+import java.util.*;
```

Comment:
Why import *? That will build dependency on everything in there.

Reply:
Yes I changed it, this time I am just importing the libraries which are required rather than importing all ('*'). Kindly let me know if assert statement I have put is what you were expecting. Thank you for the feedback.